### PR TITLE
Styling changes

### PR DESCRIPTION
**Pull request by Els-Sparta**
Styling changes

### CODE REVIEW

File: mazes/firstlevel.html [hunk 1]
```diff
@@ -13,13 +14,9 @@
...
       <div class="info">
-        <button id="start" type="button">Start</button>
-        <div class="button" id="homepage">
-          <a href="../index.html"></a>
-        </div>
-        <div class="button" id="level2">
-          <a href="secondlevel.html"></a>
-        </div>
+        <button id="start" class="btn btn-default">Start</button>
+          <a id="button1" class="btn btn-default" href="../index.html">Back to Homepage</a>
```
Comment (AEJackson) [inline mazes/firstlevel.html]: check indentation here

File: mazes/firstlevel.html [hunk 2]
```diff
@@ -13,13 +14,9 @@
...
+          <a id="button2" class="btn btn-default" href="secondlevel.html">Continue to next Maze...</a>
       </div>
         <div class="time">0</div>
```
Comment (AEJackson) [inline mazes/firstlevel.html]: check indentation here

File: mazes/instruction.html [hunk 3]
```diff
@@ -5,18 +5,22 @@
     <script
     src="https://code.jquery.com/jquery-3.3.1.min.js" integrity="sha256-FgpCb/KJQlLNfOu91ta32o/NMZxltwRo8QtmkMRdAu8=" crossorigin="anonymous"></script>
     <script type="text/javascript" src=""></script>
+        <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/css/bootstrap.min.css" integrity="sha384-BVYiiSIFeK1dGmJRAkycuHAHRg32OmUcww7on3RYdg4Va+PmSTsz/K68vbdEjh4u" crossorigin="anonymous">
```
Comment (AEJackson) [inline mazes/instruction.html]: check indentation here

File: mazes/secondlevel.html [hunk 4]
```diff
@@ -5,6 +5,7 @@
     <script
     src="https://code.jquery.com/jquery-3.3.1.min.js" integrity="sha256-FgpCb/KJQlLNfOu91ta32o/NMZxltwRo8QtmkMRdAu8=" crossorigin="anonymous"></script>
     <script type="text/javascript" src="../js/level2.js"></script>
+        <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/css/bootstrap.min.css" integrity="sha384-BVYiiSIFeK1dGmJRAkycuHAHRg32OmUcww7on3RYdg4Va+PmSTsz/K68vbdEjh4u" crossorigin="anonymous">
```
Comment (AEJackson) [inline mazes/secondlevel.html]: check indentation here

File: mazes/secondlevel.html [hunk 5]
```diff
@@ -13,9 +14,9 @@
...
       <h2 id="player"></h2>
       <div class="info">
-        <button id="start" type="button">Start</button>
-        <a href="../index.html" id="homepage"></a>
-        <a href="thirdlevel.html" id="level2"></a>
+        <button id="start" class="btn btn-default">Start</button>
+        <a id="button1" class="btn btn-default" href="../index.html">Back to Homepage</a>
+        <a id="button2" class="btn btn-default" href="thirdlevel.html">Continue to next Maze...</a>
       </div>
         <div class="time">0</div>
```
Comment (AEJackson) [inline mazes/secondlevel.html]: check indentation here

**Review by AEJackson — CHANGES_REQUESTED** [review]

**Comment by Els-Sparta** [comment]
Fixed indentations

File: mazes/firstlevel.html [hunk 6]
```diff
@@ -15,8 +15,8 @@ <h1>Maze 1</h1>
       <h2 id="player"></h2>
       <div class="info">
         <button id="start" class="btn btn-default">Start</button>
-          <a id="button1" class="btn btn-default" href="../index.html">Back to Homepage</a>
-          <a id="button2" class="btn btn-default" href="secondlevel.html">Continue to next Maze...</a>
+        <a id="button1" class="btn btn-default" href="../index.html">Back to Homepage</a>
+        <a id="button2" class="btn btn-default" href="secondlevel.html">Continue to next Maze...</a>
       </div>
         <div class="time">0</div>
```
Comment (AEJackson) [inline mazes/firstlevel.html]: check indentation here

**Review by AEJackson — CHANGES_REQUESTED** [review]

File: mazes/firstlevel.html [hunk 7]
```diff
@@ -13,13 +14,9 @@
...
-          <a href="../index.html"></a>
-        </div>
-        <div class="button" id="level2">
-          <a href="secondlevel.html"></a>
-        </div>
+        <button id="start" class="btn btn-default">Start</button>
+        <a id="button1" class="btn btn-default" href="../index.html">Back to Homepage</a>
+        <a id="button2" class="btn btn-default" href="secondlevel.html">Continue to next Maze...</a>
       </div>
         <div class="time">0</div>
```
Comment (AEJackson) [inline mazes/firstlevel.html]: check indent

**Review by AEJackson — CHANGES_REQUESTED** [review]

File: mazes/secondlevel.html [hunk 8]
```diff
@@ -18,7 +18,7 @@ <h2 id="player"></h2>
         <a id="button1" class="btn btn-default" href="../index.html">Back to Homepage</a>
         <a id="button2" class="btn btn-default" href="thirdlevel.html">Continue to next Maze...</a>
       </div>
-        <div class="time">0</div>
+      <div class="time">0</div>
         <h2 id="win"></h2>
```
Comment (AEJackson) [inline mazes/secondlevel.html]: these h2's need to be aligned too

File: mazes/firstlevel.html (hunk 7 above)
Comment (AEJackson) [reply mazes/firstlevel.html]: this doesn't seem to be done

File: mazes/secondlevel.html [hunk 9]
```diff
@@ -13,11 +14,11 @@
...
-        <button id="start" type="button">Start</button>
-        <a href="../index.html" id="homepage"></a>
-        <a href="thirdlevel.html" id="level2"></a>
+        <button id="start" class="btn btn-default">Start</button>
+        <a id="button1" class="btn btn-default" href="../index.html">Back to Homepage</a>
+        <a id="button2" class="btn btn-default" href="thirdlevel.html">Continue to next Maze...</a>
       </div>
-        <div class="time">0</div>
+      <div class="time">0</div>
         <h2 id="win"></h2>
```
Comment (AEJackson) [inline mazes/secondlevel.html]: these h2's need to be aligned

**Comment by Els-Sparta** [comment]
Fixed indentation

File: mazes/thirdlevel.html [hunk 10]
```diff
@@ -13,8 +14,8 @@
       <h1>Maze 3</h1>
       <h2 id="player"></h2>
       <div class="info">
-        <button id="start" type="button">Start</button>
-        <a href="../index.html" id="homepage"></a>
+        <button id="start" class="btn btn-default">Start</button>
+        <a id="button1" class="btn btn-default" href="../index.html">Back to Homepage</a>
       </div>
         <div class="time">0</div>
```
Comment (AEJackson) [inline mazes/thirdlevel.html]: the following lines need to be aligned

**Review by AEJackson — CHANGES_REQUESTED** [review]

**Comment by Els-Sparta** [comment]
Added more information to README.md

**Review by AEJackson — APPROVED** [review]

**Pull request merged**